dispatcher/immutable.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var Dispatcher = require('.');
+
+function get(object, keypath) {
+  return object.getIn(keypath);
+}
+
+function set(object, keypath, value) {
+  return object.setIn(keypath, value);
```

Comment:
Is that protocol supported in every immutable implementation or is that specific to immutable.js?